test/e2e-variables.env

```diff
@@ -2,6 +2,8 @@ NODE_ENV=development
 CONTENTFUL_SPACE_ID=qz0n5cdakyl9
 CONTENTFUL_DELIVERY_TOKEN=580d5944194846b690dd89b630a1cb98a0eef6a19b860ef71efc37ee8076ddb8
 CONTENTFUL_PREVIEW_TOKEN=e8fc39d9661c7468d0285a7ff949f7a23539dd2e686fcb7bd84dc01b392d698b
+CONTENTFUL_DELIVERY_API_HOST=cdn.contentful.com
```

Comment:
you can merge them in one vars `CONTENTFUL_HOST`, and it gets overridden by local configs or command line vars. WDYT?